MaterialDesignThemes.Wpf/DataGridAssist.cs

```diff
@@ -136,6 +137,17 @@ public static void SetCellPadding(DataGrid element, Thickness value)
             => element.SetValue(CellPaddingProperty, value);
         #endregion
 
+        #region AttachedProperty : SelectedCellBorderBrushProperty
+        public static readonly DependencyProperty SelectedCellBorderBrushProperty
+            = DependencyProperty.RegisterAttached("SelectedCellBorderBrush", typeof(Brush), typeof(DataGridAssist),
+                new FrameworkPropertyMetadata(null, FrameworkPropertyMetadataOptions.Inherits));
```

Comment:
It doesn't look like we need this property to inherit down, so there is a small performance gain by removing it
```suggestion
                new PropertyMetadata(null));
```